builtin/logical/pki/cert_util.go

```diff
@@ -856,8 +865,16 @@ func createCertificate(creationInfo *creationBundle) (*certutil.ParsedCertBundle
 	}
 
 	if creationInfo.SigningBundle != nil {
-		result.IssuingCABytes = creationInfo.SigningBundle.CertificateBytes
-		result.IssuingCA = creationInfo.SigningBundle.Certificate
+		certPath := creationInfo.SigningBundle.GetCertificatePath()
+		result.IssuingCABytes = certPath[0].Bytes
+		result.IssuingCA = certPath[0].Certificate
```

Comment:
For safety, can you ensure that the length of `certPath` is at least one here? Better to return an error than panic.